doc/api/process.md

```diff
@@ -2426,8 +2426,16 @@ flag's behavior.
 ## `process.umask([mask])`
 <!-- YAML
 added: v0.1.19
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/32499
+    description: Calling `process.umask()` with no arguments is deprecated.
+
 -->
 
+> Stability: 0 - Deprecated. Calling `process.umask()` with no arguments is
+> deprecated. No alternative is provided.
```

Comment:
The stability setting isn't right, is it? The stability applies to the method, not a particular use of the method. That will make it look like the method is deprecated itself, but it's not. We don't do this for any other methods. I'm going to open a PR to change this to a paragraph rather than a stability banner.

Reply:
We could provide separate entries for `process.umask()` and `process.umask(mask)` :woman_shrugging:

Reply:
Sure, that could work too.

Reply:
https://github.com/nodejs/node/pull/32711